composer.json

```diff
@@ -34,7 +34,7 @@
         "rector/rector": "^1.0",
         "roave/infection-static-analysis-plugin": "^1.16",
         "spatie/phpunit-watcher": "^1.23",
-        "vimeo/psalm": "^4.30|^5.20",
+        "vimeo/psalm": "^4.30|^5.24",
```

Comment:
```suggestion
        "vimeo/psalm": "^5.24",
```

Reply:
Tried this way (the first commit). It is cause of errors in db drivers when composer install.
Will be changed after PHP version is raised to 8.1 for all db drivers.